src/NotificationEventMailing.php

```diff
@@ -326,11 +325,11 @@ public static function send(array $data)
 
                                 //find width
                                 $custom_width = null;
-                                if (preg_match("/width=[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $wmatches)) {
+                                if (preg_match("/width(?:=|&#61;)[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $wmatches)) {
                                     $custom_width = intval($wmatches[1]);
                                 }
                                 $custom_height = null;
-                                if (preg_match("/height=[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $hmatches)) {
+                                if (preg_match("/height(?:=|&#61;)[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $hmatches)) {
```

Comment:
Both of these changes are not correct. The `=` is only encoded inside attributes values themselves, see the code snippet in your PR description: `width="4032"`.

Reply:
you're right